lib/SPIRV/SPIRVReader.cpp

```diff
@@ -470,7 +470,7 @@ Type *SPIRVToLLVM::transType(SPIRVType *T, bool IsClassMember) {
 
   SPIRVDBG(spvdbgs() << "[transType] " << *T << " -> ";)
   T->validate();
-  switch (T->getOpCode()) {
+  switch (static_cast<size_t>(T->getOpCode())) {
```

Comment:
Minor: Why `size_t`? AFAIK on most systems `size_t` has 64-bit width, but 32 bits should be enough for OpCode.